README.md

```diff
@@ -35,6 +35,17 @@ they are not discussed elsewhere in this document.
 
 - EMAILER_RECIPIENT_OVERRIDE: If set, `emailer` will send all non-test email to this email address.
 
+## AWS configuration setup for the storage
+
+- SIMPLIFIED_AWS_S3_BUCKET_NAME (mandatory): The name of the bucket to use on S3
+- SIMPLIFIED_AWS_ACCESS_KEY_ID: The AWS access key ID
+- SIMPLIFIED_AWS_SECRET_ACCESS_KEY: The AWS secret key
+- SIMPLIFIED_AWS_S3_ENDPOINT_URL: The API endpoint for the S3 bucket
+- SIMPLIFIED_AWS_PROFILE: The AWS credentials file profile name
```

Comment:
This is the case right now, everything except the bucket name is optional. Since boto3 is in use, it will authenticate in the order described in the link

Reply:
What I'm wondering is if we need to define our own environment variables for:
- SIMPLIFIED_AWS_ACCESS_KEY_ID
- SIMPLIFIED_AWS_SECRET_ACCESS_KEY
- SIMPLIFIED_AWS_PROFILE

Since you could just use the ones that Boto3 defines without any code on our side. Like you could use these instead, and not need any code on our side to handle it:
- AWS_ACCESS_KEY_ID
- AWS_SECRET_ACCESS_KEY
- AWS_PROFILE

And have the same result as setting our `SIMPLIFIED_AWS_*` environment variables. It does look like Boto3 doesn't have any env variable for endpoint URL. So it seems like we would need to keep:

- SIMPLIFIED_AWS_S3_BUCKET_NAME
- SIMPLIFIED_AWS_S3_ENDPOINT_URL (optional)

Reply:
Yes, we could do that, the reason for having our own env names was for encapsulation purposes (no confusion between this and any other app), but I guess this is not required in a docker env. I'll make the change